ddln_imaginator.proto

```diff
@@ -21,6 +21,64 @@ enum ImageFormat {
     GRAY = 4;
 }
 
+message Environment {
+    message Clouds {
+        enum Type {
+            NONE = 0;
+            ALTOCUMULUS = 1;
+            CLOUDS_BASE = 2;
```

Comment:
CLOUDS_BASE does not sound like a meteorological class. Is there another name for this type?

Reply:
Name is directly from Unigine API. I don't know the type of this cloud but it's a default cloud when you create a 'ObjectCloudLayer'.